Makefile

```diff
@@ -5,7 +5,6 @@ ARCH_FLAGS=-arch armv7 -arch armv7s -arch arm64
 
 LDFLAGS	=\
 	-F$(SDK)/System/Library/Frameworks/\
-	-F$(SDK)/System/Library/PrivateFrameworks/\
```

Comment:
This wouldn't build on my machine unless I removed this no longer present directory (on my machine at least). It seemed to build fine once I did.